airflow/api_connexion/endpoints/role_and_permission_endpoint.py

```diff
@@ -68,11 +68,19 @@ def get_roles(*, order_by: str = "name", limit: int, offset: Optional[int] = Non
     appbuilder = current_app.appbuilder
     session = appbuilder.get_session
     total_entries = session.query(func.count(Role.id)).scalar()
+    direction = "desc" if order_by.startswith("-") else "asc"
```

Comment:
Rather than the double `getattr` we should use the standalone asc/desc functions https://docs.sqlalchemy.org/en/13/core/sqlelement.html?highlight=desc#sqlalchemy.sql.expression.desc

```suggestion
    direction = desc if order_by.startswith("-") else asc
```

etc